Mail2Bug/Email/IIncomingEmailMessage.cs

```diff
@@ -15,7 +15,13 @@ public interface IIncomingEmailMessage
         string Subject { get; }
         string RawBody { get; }
         string PlainTextBody { get; }
+
+        // Instead of using the whole ConversationIndex, use the ConversationID, which is
+        // the guid portion of the ConversationIndex, bytes 6 through 22. The ConversationID does not
+        // contain timestamps or additional reply information.
+        // See https://msdn.microsoft.com/en-us/library/ee202481(v=exchg.80).aspx for more information
```

Comment:
We should avoid adding this new property to IIncomingEmailMessage, since the distinction is very Exchange specific
Instead, EWSIncomingMailMessage should expose two (public) functions that provide each of these, and ConversationIndex property (possibly renamed) should return one or the other based on the config.